packages/core/src/payment/payment.ts

```diff
@@ -195,6 +197,28 @@ interface AdyenV2Card {
     bigpay_token?: void;
 }
 
+type BlueSnapDirectEcpAccountType =
+    | 'CONSUMER_CHECKING'
+    | 'CONSUMER_SAVINGS'
+    | 'CORPORATE_CHECKING'
+    | 'CORPORATE_SAVINGS';
+
+export interface BlueSnapDirectEcpInstrument {
```

Comment:
This type is provider-specific, can we rename it to something generic such as `BankTransferInstrument` or something?